### PR TITLE
[DOCS-7518] Add note about security levels in Salesforce Connector app

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

**Review by jottley — APPROVED** [review]

File: salesforce/latest/install/index.md
Comment (ariadnags) [inline salesforce/latest/install/index.md:1]: "Selecting a security level can have an affect on future usage."
I think it's supposed to be an effect?

**Review by ariadnags — CHANGES_REQUESTED** [review]
"Selecting a security level can have an affect on future usage."
I think it's supposed to be an effect?

**Comment by anxumalo** [comment]
> "Selecting a security level can have an affect on future usage." I think it's supposed to be an effect?

@ariadnags Good point ..  we have a choice of using either "_can affect.._."  or "_have an effect on .._". I'll use the latter.

**Review by ariadnags — APPROVED** [review]
LGTM

**Pull request merged**